### DISCUSSION TITLE
Tips: Raw

### DISCUSSION THREAD

**Discussion by rikardronnkvist** [jonkpirateboy/Pass-fur-alle#106] [Ideas]
På passfuralle.se kan man länka direkt till en rå variant JS-filen istället för zippen...  blir nog enklare för en novis:

https://raw.githubusercontent.com/jonkpirateboy/Pass-fur-alle/main/pass-fur-alle.js

**Comment by jonkpirateboy** [comment]
Jag tänker att det är enklare att länka till zippen eftersom vissa antivirus-program säkert kan reagera på att man försöker ladda hem en js-fil.

**Reply by rikardronnkvist** [reply]
Inte riktigt så det fungerar med raw-siten, den visar JS-filen som text direkt i webläsaren...

**Comment by ghost** [comment]
Halloj, 

tack för ett fantastiskt initiativ. Fungerar scriptet fortsatt idag? Jag drog igång det igår och det har jobbat i 20 timmar nu kanske. 

Tack på förhand! 

Mvh, Adam

**Reply by jonkpirateboy** [reply]
Det funkar fortfarande, men det bör ha stannat en hel del gånger på 20 timmar för captcha.
